packages/plugin-ext/src/main/browser/view/tree-view-widget.tsx

```diff
@@ -280,7 +280,18 @@ export class TreeViewWidget extends TreeViewWelcomeWidget {
             title
         });
 
-        const children = this.getCaption(node);
+        const children = [this.getCaption(node)];
```

Comment:
Using `this.getCaption(node)` and `this.toNodeName(node)` will lead to duplicated labels. Please remove the `this.getCaption(node)`, as its not necessary anymore here.

```suggestion
        const children: React.ReactNode = [];
```

Reply:
@shuyaqian Oops, I was a bit sleepy yesterday, when I suggested this. The brackets `[]` are missing right behind `React.ReactNode`. Also, would you mind squashing your commits, so that the commit history only contains a single commit from you? Currently, my suggestions are also in there.